hmcts/pre-api
# Fix: MK Final Assets Not Playable

### JIRA ticket(s)
- S28-3179


### Change description
- Fix issue causing v1 asset to be unplayable by MK





> [!IMPORTANT]
> This PR requires manual testing by QA. Please notify the QA team @hmcts/pre-rec-evidence-qa.

**QA instructions**
- Test: Start a live event, start CVP recording, end live event. On the MK dashboard, see two jobs fired off (EncodeFromIngest and EncodeFromMp4).  Once both complete, got to asset created by EncodeFromMp4 job, add a streaming locator (clear streaming) and see after a few seconds the asset is playable. See capture session marked as RECORDING_AVAILABLE
-Test: Start a live event, DO NOT start CVP recording, end live event. On the MK dashboard, see one job fired off (EncodeFromIngest).  See capture session marked as NO_RECORDING
